<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFriendList</class>
 <widget class="QWidget" name="CFriendList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lb_name">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lb_icon">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>icon</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lb_signed">
     <property name="text">
      <string>这个家伙很懒，什么也没留下</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_addFriend">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/addFriend/a3f92ae9_E771320_0737bafa.png</normaloff>:/addFriend/a3f92ae9_E771320_0737bafa.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tbWidget_friend">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tb_online">
      <attribute name="title">
       <string>在线好友</string>
      </attribute>
      <widget class="QScrollArea" name="sca_online">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>6</y>
         <width>430</width>
         <height>431</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="sca_inOnline">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>400</width>
          <height>429</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QWidget" name="wdg_online" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <zorder>verticalSpacer</zorder>
           <zorder>verticalSpacer</zorder>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>375</width>
             <height>346</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tb_offline">
      <attribute name="title">
       <string>离线好友</string>
      </attribute>
      <widget class="QScrollArea" name="sca_offline">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>6</y>
         <width>430</width>
         <height>431</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>485</width>
         <height>491</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="sca_inOffline">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>400</width>
          <height>429</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="wdg_offline" native="true">
           <property name="minimumSize">
            <size>
             <width>406</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>406</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>406</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
